### PR TITLE
Cleanups before `session-dbus` branch

### PR DESCRIPTION
**Pull request by ndim**
These are a few cleanups which I happend upon while working on the `session-dbus` branch, but which are not related to `session-dbus` at all.

Each commit should speak for itself.

### CODE REVIEW

**Comment by ndim** [comment]
As far as I am concerned, these are ready to be merged into (at your option) either

  * the `release` branch after which the `session-dbus` branch can be fast forwarded to `release`
  * the `session-dbus` branch directly

These are preparatory work for the `session-dbus` branch, rearranging things to make later changes easier, and the like. However, they should also not make `release` worse, so they could go in there as well. Your call.

File: soundcraft/notepad.py [hunk 1]
```diff
@@ -56,7 +49,7 @@ def __init__(
             except Exception:
                 # Fall-back to class name, since reading the product over USB requires write access
                 self.product = self.__class__.__name__
-            self.fwVersion = f"{major}.{minor}"
+            self.fwVersion = "%d.%02d" % (major, minor)
```
Comment (lack) [inline soundcraft/notepad.py:52]: We can still use f-strings for this:

```
self.fwVersion = f"{major}.{minor:02d}"
```

Comment (ndim) [reply soundcraft/notepad.py:52]: Some things are both less than obvious and less than well documented :-/

Anyways, do you
  * want me to force-push a changed commit which uses an f string
  * want me to add another commit which fixes this to use an f string
  * want to fix it yourself
?

Comment (ndim) [reply soundcraft/notepad.py:52]: Sorry, when `pre-commit` is running `pytest`, it still produces this:

```
>           self.fwVersion = f"{major}.{minor:02d}"
E           TypeError: unsupported format string passed to MagicMock.__format__

soundcraft/notepad.py:65: TypeError
```

This appears to mean the issue is somewhere inside MagicMock, i.e. somewhere in the Python runtime, which I have no intention to fix at this time.

Comment (lack) [reply soundcraft/notepad.py:52]: Agreed.  I'm happy with this as-is then :)

**Comment by lack** [comment]
By the way, forgot to mention:  This is all awesome.  Thanks so much for all the help!!

**Pull request merged**